subprojects/libero-distribution/README.md

```diff
@@ -108,4 +166,4 @@ I Release You!
    versions:
      release: "1.2.3"
      development: "1.2.4-SNAPSHOT"
```

Comment:
Do you think we should use a replace variable here just to show how they work, e.g., `release: "1.2.3.${timestamp}"`?